Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.yml

```
@@ -2631,6 +2654,149 @@ script:
+    - name: site_ids
+      description: A comma-separated list of site IDs.
+      isArray: true
+    description: Returns all installed singularity marketplace applications that match the specified filter values.
+    name: sentinelone-list-installed-singularity-marketplace-applications
+    outputs:
+    - contextPath: SentinelOne.InstalledApps.ID
+      description: The application ID.
+      type: string
+    - contextPath: SentinelOne.InstalledApps.Account
+      description: The account name.
+      type: string
+    - contextPath: SentinelOne.InstalledApps.AccountId
+      description: The account ID.
+      type: string
+    - contextPath: SentinelOne.InstalledApps.ApplicationCatalogId
+      description: The application Catalog ID.
+      type: string
+    - contextPath: SentinelOne.InstalledApps.ApplicationCatalogName
+      description: The application Catalog name.
+      type: string
+    - contextPath: SentinelOne.InstalledApps.AlertMessage
+      description: The alert message.
+      type: string
+    - contextPath: SentinelOne.InstalledApps.CreatedAt
+      description: Application created at.
+      type: date
+    - contextPath: SentinelOne.InstalledApps.Creator
+      description: Application creator.
+      type: string
+    - contextPath: SentinelOne.InstalledApps.CreatorId
+      description: Application creator ID.
+      type: string
+    - contextPath: SentinelOne.InstalledApps.DesiredStatus
+      description: Application desired status.
+      type: string
+    - contextPath: SentinelOne.InstalledApps.HasAlert
+      description: Application has alert.
+      type: boolean
+    - contextPath: SentinelOne.InstalledApps.LastEntityCreatedAt
+      description: Application last entity created at.
+      type: date
+    - contextPath: SentinelOne.InstalledApps.Modifier
+      description: Modifier.
+      type: string
+    - contextPath: SentinelOne.InstalledApps.ModifierId
+      description: Modifier ID.
+      type: string
+    - contextPath: SentinelOne.InstalledApps.ScopeId
+      description: The scope ID.
+      type: string
+    - contextPath: SentinelOne.InstalledApps.ScopeLevel
+      description: The scope level.
+      type: string
+    - contextPath: SentinelOne.InstalledApps.Status
+      description: Status of application.
+      type: string
+    - contextPath: SentinelOne.InstalledApps.UpdatedAt
+      description: Application updated at.
+      type: string
+    - contextPath: SentinelOne.InstalledApps.ApplicationInstanceName
+      description: Application instance name.
+      type: string
+  - arguments:
+    - name: account_ids
+      description: A comma-separated list of account IDs.
+      isArray: true
+    - name: role_ids
+      description: A comma-separated list of rbac roles to filter by.
+      isArray: true
+    - name: ids
+      description: A comma-separated list of service user IDs to filter by.
+      isArray: true
+    - name: site_ids
```

According to our conventions
```suggestion
    - name: site_id
```